File: Problems_and_Solutions/days-10 to 19/day-14/Java/Solution1.java
class Solution1 {
    public static void print_divisors(int n) {
        for(int i=1; i<=n; i++){
            if(n%i == 0){
                System.out.print(i + " ");
            }
        }
    }

    public static void main(String[] args) {
        int n = 20;
        print_divisors(n);
    }
}

File: Problems_and_Solutions/days-10 to 19/day-14/c++/solution 1.cpp
class Solution {
  public:
    void print_divisors(int n) {
        // Code here.
        if(n!=1)
            cout<<1<<" ";
        for(int i=2;i<=n/2;i++)
            if(n%i==0)
                cout<<i<<" ";
        cout<<n<<" ";
    }
};